File: PB_Online_Exam_2021_October/PC_Store.py
cpu_in_dollars = float(input())
gpu_in_dollars = float(input())
ram_in_dollars = float(input())
num_of_ram = int(input())
discount = float(input())

cpu_in_leva = cpu_in_dollars * 1.57
gpu_in_leva = gpu_in_dollars * 1.57
ram_in_leva = ram_in_dollars * 1.57
total_ram = ram_in_leva * num_of_ram

cpu_after_discount = cpu_in_leva - (cpu_in_leva * discount)
gpu_after_discount = gpu_in_leva - (gpu_in_leva * discount)

result = cpu_after_discount + gpu_after_discount + total_ram

print(f'Money needed - {result:.2f} leva.')

File: C++/Basics/Exam/FirstCode/PCStore.cpp
#include <iostream>
using namespace std;

int main()
{
	double cpuInUsd, gpuInUsd, ramInUsd;
	int numOfRam;
	double percentDiscount;

	cin >> cpuInUsd >> gpuInUsd >> ramInUsd >> numOfRam >> percentDiscount;

	double cpuInLv = cpuInUsd * 1.57;
	double gpuInLv = gpuInUsd * 1.57;
	double ramInLv = ramInUsd * 1.57;

	double totalRam = numOfRam * ramInLv;

	double cpuAfterDiscount = cpuInLv - (cpuInLv * percentDiscount);
	double gpuAfterDiscount = gpuInLv - (gpuInLv * percentDiscount);

	double totalSum = cpuAfterDiscount + gpuAfterDiscount + totalRam;

	cout.setf(ios::fixed);
	cout.precision(2);

	cout << "Money needed - " << totalSum << " leva." << endl;
}